src/Evolution/DiscontinuousGalerkin/Limiters/WenoOscillationIndicator.cpp

```diff
@@ -163,6 +165,54 @@ Matrix compute_indicator_matrix(
   return result;
 }
 
+// Helper function for caching a matrix that depends on an Index.
```

Comment:
I think `Index` in this comment and in the function name is a bit confusing since with "mesh index" I would associate a particular point on the grid. I suggest `extents`.